src/styles/blocks/main.scss

```diff
@@ -0,0 +1,55 @@
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
```

Comment:
usually filename should match the BEM block name (calendar.scss in this case)
And `.body` is also a separate BEM block

Just FYI